packages/xo-web/src/xo-app/home/index.js

```diff
@@ -715,7 +715,12 @@ export default class Home extends Component {
         }
         const backedUpVms = uniq(
           flatMap(
-            filter(jobs, job => some(schedulesByJob[job.id], schedule => schedule.enabled)),
+            createSelector(
+              () => jobs,
+              () => schedulesByJob,
+              (_jobs, _schedulesByJob) =>
+                filter(_jobs, job => some(_schedulesByJob[job.id], schedule => schedule.enabled))
+            )(),
```

Comment:
A selector is a memoized function, it does not provide any benefits if it's only called once :slightly_smiling_face: